static/app/components/events/interfaces/frame/openInContextLine.tsx

```diff
@@ -54,30 +58,58 @@ function OpenInContextLine({lineNo, filename, components}: Props) {
 
 export {OpenInContextLine};
 
-const OpenInContainer = styled('div')<{columnQuantity: number}>`
+const fadeIn = keyframes`
+  from { opacity: 0; }
+  to { opacity: 1; }
+`;
+
+const OpenInContainer = withOrganization(styled('div')<{
+  columnQuantity: number;
+  organization: Organization;
+}>`
   display: flex;
   gap: ${space(1)};
   align-items: center;
   z-index: 1;
-  color: ${p => p.theme.subText};
-  background-color: ${p => p.theme.background};
   font-family: ${p => p.theme.text.family};
-  border-bottom: 1px solid ${p => p.theme.border};
-  padding: ${space(0.25)} ${space(3)};
-  box-shadow: ${p => p.theme.dropShadowLight};
   text-indent: initial;
   overflow: auto;
   white-space: nowrap;
-`;
+  ${p =>
+    p.organization.features.includes('issue-details-stacktrace-link-in-frame')
+      ? `
+   color: ${p.theme.linkColor};
+   animation: ${fadeIn} 0.2s ease-in-out forwards;
+   padding: ${space(0)} ${space(0)};
+   `
+      : `
+  color: ${p.theme.subText};
+  background-color: ${p.theme.background};
+  border-bottom: 1px solid ${p.theme.border};
+  padding: ${space(0.25)} ${space(3)};
+  box-shadow: ${p.theme.dropShadowLight};
+   `}
+`);
 
-const OpenInLink = styled(ExternalLink)`
+const OpenInLink = withOrganization(styled(ExternalLink)<{organization: Organization}>`
```

Comment:
Because the stack trace can be rendered inside the share view, organization context can be null so I believe this will error out in that case